docs/guide/using_existing_contracts.rst

```diff
@@ -61,7 +61,7 @@ Automatic fee estimation
 
 For testing purposes it is possible to enable automatic fee estimation when making
 a transaction. Starknet.py will then use ``estimate_fee()`` internally and use value
-returned by it multiplied by ``1.1`` as a ``max_fee``.
+returned by it multiplied by ``1.5`` as a ``max_fee``.
```

Comment:
Would add information about adjusting the default multiplier here or somewhere else.

Reply:
Adjusting? You mean the fact that it has been changed or? It cannot be adjusted manually

Reply:
I'd like it to be adjustable by the user. That's why I wanted the  `ESTIMATED_FEE_MULTIPLIER` to be a part of Account now.